lib/three.js

```diff
@@ -6,10 +6,23 @@ if (THREE.TextureLoader) {
 }
 
 // TODO: Eventually include these only if they are needed by a component.
-THREE.OBJLoader = require('../lib/vendor/OBJLoader')(THREE);
-THREE.ColladaLoader = require('../lib/vendor/ColladaLoader')(THREE);
-THREE.Raycaster = require('../lib/vendor/Raycaster')(THREE);
-THREE.VRControls = require('../lib/vendor/VRControls');
-THREE.VREffect = require('../lib/vendor/VREffect');
 
-module.exports = THREE;
+// THREE.OBJLoader
+require('../node_modules/three-dev/examples/js/loaders/OBJLoader');
+
+// THREE.ColladaLoader
+require('../node_modules/three-dev/examples/js/loaders/ColladaLoader');
+
+// THREE.ColladaLoader
+require('../node_modules/three-dev/examples/js/loaders/ColladaLoader');
+
+// THREE.Raycaster
+require('../lib/vendor/Raycaster');
```

Comment:
@ngokevin if you can change your `three-dev`'s `package.json` the `directories` to include `src/core`, we could reference `Raycaster.js` so we wouldn't need it here.